css/build/style.css

```diff
@@ -0,0 +1,215 @@
+/* general */
+body {
+  min-width: 1366;
+  margin: 0;
+  padding: 0;
+}
+
+a {
+  text-decoration: none;
+}
+
+h2 {
+  padding: 0;
+  margin: 0;
+}
+
+p {
+  padding: 0;
+  margin: 0;
+}
+
+/* typography */
+.primary-font-family {
+  font-family: "Poppins", sans-serif;
+}
+
+.logo-font {
+  font-family: "Poppins", sans-serif;
+  font-size: 18px;
+  font-weight: 700;
+  color: #6070ff;
+}
+
+.banner-header {
+  font-family: "Poppins", sans-serif;
+  font-size: 38px;
+  line-height: 52px;
+  font-weight: 700;
+  color: #172b4d;
+}
+
+.body-text-1 {
+  font-family: "Poppins", sans-serif;
+  font-size: 16px;
+  line-height: 52px;
+  font-weight: 300;
+  color: #172b4d;
+}
+
+.body-text-2 {
+  font-family: "Poppins", sans-serif;
+  font-size: 15px;
+  line-height: 24px;
+  font-weight: 300;
+  color: #172b4d;
+}
+
+.head-text-1 {
+  font-family: "Poppins", sans-serif;
+  font-size: 16px;
+  line-height: 24px;
+  font-weight: 400;
+  color: #6070ff;
+}
+
+/* colors */
+.primary-color {
+  color: #6070ff !important;
+}
+
+.secondary-color {
+  color: #172b4d;
+}
+
+.background-color {
+  background-color: #e5e5e5;
+}
+
+/* header */
+.nav-bar {
+  position: fixed;
+  top: 0;
+  z-index: 100;
+  background-color: white;
+  width: calc(100% - 40px);
+  display: flex;
+  padding-top: 0;
+  padding-left: 20px;
+  padding-right: 20px;
+}
+
+.nav-bar ul {
+  list-style-type: none;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0;
+  width: 100%;
+}
+
+.nav-bar li {
+  text-decoration: none;
+}
+
+.menu-icon {
+  color: #6070ff;
+  padding: 10px;
+  flex: 1;
+}
+
+.nav-logo {
+  flex: 3;
+}
+
+/* banner */
+.banner {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  background-color: white;
+  min-height: 100vh;
+  border-radius: 0 0 0 70px;
+}
+
+.banner-content {
+  margin-top: 20px;
+}
+
+.banner-stack {
+  position: absolute;
+  width: calc(100% - 40px);
+  padding-top: 0;
+  padding-left: 20px;
+  padding-right: 20px;
+}
+
+.banner-background {
+  position: absolute;
+  height: 100vh;
+  width: 100%;
+}
+
+.banner-social-icon {
+  color: #172b4d;
+  padding: 0 10px 0 0;
+}
+
+.banner-socials {
+  display: flex;
+  padding: 10px 0 0 0;
+  margin: 0;
+}
+
+.banner-socials li {
+  list-style-type: none;
+  display: inline;
+}
+
+.vector1 {
+  position: absolute;
+  bottom: 20px;
+  right: 110px;
+}
+
+.vector2 {
+  position: absolute;
+  bottom: 100px;
+  right: 50px;
+}
+
+.vector3 {
+  position: absolute;
+  left: 0;
+  top: 70px;
+}
+
+.vector4 {
+  position: absolute;
+  top: 0;
+}
+
+.vector5 {
+  position: absolute;
+  height: 100px;
+  top: 65px;
+  right: 0;
+}
+
+.ellipse1 {
+  position: absolute;
+  bottom: 45px;
+  left: 50px;
+}
+
+.ellipse2 {
+  position: absolute;
+  top: 80px;
+  right: 100px;
+}
+
+.socials-icons {
+  height: 30px;
+  width: 30px;
+  color: blue;
+  margin-right: 10px;
+}
```

Comment:
- [ ] Kindly make sure your icons have the same **height**, **width**, **color**, and **spacing between them** as from the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) design. I noticed that your icons don't accept the **color** attribute, it is because you use them as images. You can use icons instead to fix the issue. e.g.: `<a href=""><i class="fa fa-twitter"></i></a>` then the color will work.
### Your Icons: 
![icons_1](https://user-images.githubusercontent.com/83042742/170402697-6e3cc76e-ad0d-4c40-951e-2c27084f0f98.png)
### Figma Icons: 
![figma_icon](https://user-images.githubusercontent.com/83042742/170402801-cd0de979-245e-428e-bc5e-15af6576da58.png)